### DISCUSSION TITLE
Cannot re-apply for faculty discount

### DISCUSSION THREAD

**Discussion by tpoisot** [community/community#63565] [GitHub Education]
### Hi everyone,

I have been trying to re-apply for the faculty discount for *months* (literal months), for accounts `@tpoisot` and `@poisotlab`. I opened tickets with support, that have remained unanswered.

Basically, I can fill-in the page to provide my academic credentials, allow github to access my location, but when I click "continue" _nothing_ happens. I tried with different browsers.

I'm 2 weeks from the beginning of the term and moving all my classes to gitlab, and there's a chance my group organization will follow immediately after if the discount process really is that unreliable.

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
